plugins/SitesManager/SitesManager.php

```diff
@@ -68,6 +68,19 @@ public function redirectDashboardToWelcomePage(&$module, &$action)
             return;
         }
 
+        if (self::isSiteEmpty($siteId)) {
+            $session = new SessionNamespace('siteWithoutData');
+            if (!empty($session->ignoreMessage)) {
+                return;
+            }
+
+            $module = 'SitesManager';
+            $action = 'siteWithoutData';
+        }
+    }
+
+    public static function isSiteEmpty($siteId)
```

Comment:
Would we maybe name this `hasTrackedAnyTraffic` to be more clear?

Reply:
Wonder if it could be even moved out of the plugin class since it is more like a model but not too important as it was here before already anyway.